app.tsx

```diff
@@ -104,7 +106,8 @@ const App = (props: NavigationComponentProps) => {
       Alert.alert('Conta não encontrada', errorMessage, [{ text: 'OK', onPress: () => console.log('OK Pressed') }]);
     },
   };
-  const [login] = useMutation(loginMutation);
+  const [login, { data, loading, error }] = useMutation(loginMutation);
```

Comment:
```suggestion
  const [login, { loading }] = useMutation(loginMutation);
```

Como o `data` e o `error` não são usados, podem ser omitidos